src/index.ts

```diff
@@ -38,11 +37,17 @@ import { loadUserData, saveUserData } from './storage/user-data'; // eslint-disa
 import { setupUpdates } from './updates';
 
 if ( ! isCLI() && ! process.env.IS_DEV_BUILD ) {
+	const version = app.getVersion();
+	const [ baseVersionWithBeta ] = version.split( '-dev.' );
+	const isDevBuild = version.includes( '-dev.' ) || process.env.NODE_ENV === 'development';
```

Comment:
Great idea to make dev builds submit errors to the `development` environment in Sentry 👍